library/libraries.js

```diff
@@ -993,8 +993,8 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
         icon: 'vue',
         url: 'http://vuejs.org/',
         test: function(win) {
-            if (win.Vue && win.Vue.compiler) {
-                return { version: 'N/A' };
+            if (win.Vue && win.Vue.version) {
```

Comment:
Please continue to use one of the non-`version` properties from the `Vue` object when checking for existence of `Vue` (such as one of these: http://vuejs.org/v2/api/#Global-API)

Otherwise, this is a nice update.  Thanks!